code/modules/shuttle/shuttles/escape_shuttle.dm

```diff
@@ -71,11 +75,94 @@
 		door_handler.control_doors("force-unlock")
 		return
 
+	if(prob(CRASH_LAND_PROBABILITY))
+		create_crash_point()
+	else
+		destination = null
+
 	set_mode(SHUTTLE_IGNITING)
 	on_ignition()
 	setTimer(ignitionTime)
 	launched = TRUE
 
+/obj/docking_port/mobile/escape_shuttle/proc/create_crash_point()
+	for(var/i = 1 to 10)
+		var/list/all_ground_levels = SSmapping.levels_by_trait(ZTRAIT_GROUND)
+		var/ground_z_level = all_ground_levels[1]
+
+		var/list/area/potential_areas = SSmapping.areas_in_z["[ground_z_level]"]
+
+		var/area/area_picked = pick(potential_areas)
+
+		var/list/potential_turfs = list()
+
+		for(var/turf/turf_in_area in area_picked)
+			potential_turfs += turf_in_area
+
+		if(!length(potential_turfs))
+			continue
+
+		var/turf/turf_picked = pick(potential_turfs)
+
+		var/obj/docking_port/stationary/escape_pod/crash_land/temp_escape_pod_port = new(turf_picked)
+		temp_escape_pod_port.width = width
+		temp_escape_pod_port.height = height
+		temp_escape_pod_port.id = id
+
+		if(!check_crash_point(temp_escape_pod_port))
+			qdel(temp_escape_pod_port)
+			continue
+
+		destination = temp_escape_pod_port
+		break
+
+	if(destination)
+		crash_land = TRUE
+
+/obj/docking_port/mobile/escape_shuttle/proc/check_crash_point(obj/docking_port/stationary/escape_pod/crash_land/checked_escape_pod_port)
+	for(var/turf/found_turf as anything in checked_escape_pod_port.return_turfs())
+		var/area/found_area = get_area(found_turf)
+		if(found_area.flags_area & AREA_NOTUNNEL)
+			return FALSE
+
+		if(!found_area.can_build_special)
+			return FALSE
+
+		if(istype(found_turf, /turf/closed/wall))
+			var/turf/closed/wall/found_closed_turf = found_turf
+			if(found_closed_turf.hull)
+				return FALSE
+
+		if(istype(found_turf, /turf/closed/shuttle))
+			return FALSE
+
+	return TRUE
+
+/obj/docking_port/mobile/escape_shuttle/enterTransit()
+	. = ..()
+
+	if(!crash_land)
+		return
+
+	for(var/turf/found_turf as anything in destination.return_turfs())
+		if(istype(found_turf, /turf/closed))
+			found_turf.ChangeTurf(/turf/open/floor)
+
+	cell_explosion(destination.return_center_turf(), 300, 25, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, create_cause_data("evac pod crash"))
+
+/obj/docking_port/mobile/escape_shuttle/on_prearrival()
+	. = ..()
+
+	if(!crash_land)
+		return
+
+	movement_force = list("KNOCKDOWN" = 0, "THROW" = 5)
+
+	for(var/mob/evac_mob in shuttle_areas[1])
```

Comment:
i know we're like never gonna have more than 1 area for escape pods but... 🥺 could we iterate over shuttle areas here?